vision/transforms/random_resized_crop_test.go

```diff
@@ -32,14 +32,14 @@ func TestRandomResizedCrop(t *testing.T) {
 	trans = RandomResizedCrop(20)
 	out = trans.Run(input)
 	a.Equal(20, out.Bounds().Max.X)
-	r, g, b, _ := out.At(4, 11).RGBA()
-	a.Equal(uint32(0x4242), r)
+	r, g, b, _ := out.At(7, 5).RGBA()
+	a.Equal(uint32(0x2424), r)
 	a.Equal(uint32(0x0000), g)
-	a.Equal(uint32(0xbdbd), b)
+	a.Equal(uint32(0xdbdb), b)
```

Comment:
How come these magic numbers like `0xdbdb`?

Reply:
I print and find out the pixel color since we use a fixed random seed, the color should be always the same.

Reply:
I added some bounds test, and

Reply:
The color should between blue and red, I updated this PR and check the range to avoid the magic number.